src/js/apps/admin/program/workflows/workflows_app.js

```diff
@@ -20,6 +23,32 @@ export default App.extend({
     this.actions = actions;
 
     this.showChildView('content', new ListView({ collection: actions }));
+
+    if (_DEVELOP_) {
```

Comment:
since we can, let's do `if (!_DEVELOP) return;` here so later when we pull the flag out, the change is just the one line